tests/test_root_entrypoint.py

```diff
@@ -1,3 +0,0 @@
-def test_root_entrypoint(client):
```

Comment:
I think this will close https://github.com/redhat-beyond/Hotails/issues/61

Reply:
Thats right , I have mentioned it in PR "Connected Issue/Story" section .